src/main/scala/io/iohk/ethereum/blockchain/sync/PeerListSupportNg.scala

```diff
@@ -44,6 +44,11 @@ trait PeerListSupportNg { self: Actor with ActorLogging =>
 
   def getPeerById(peerId: PeerId): Option[Peer] = handshakedPeers.get(peerId).map(_.peer)
 
+  def getPeerWithHighestBlock: Option[PeerWithInfo] =
+    peersToDownloadFrom.values.toList.sortBy { case PeerWithInfo(_, peerInfo) =>
+      peerInfo.maxBlockNumber
+    }(Ordering[BigInt].reverse).headOption
```

Comment:
nitpick: `Ordering[BigInt].reverse` could be an explicit value to save on `reverse` calls